### PR TITLE
add support for arbitrary metric matching

### PR DESCRIPTION
**Pull request by aschepis**

### CODE REVIEW

**Comment by aschepis** [comment]
@klingerf it doesn't look like this project has been touched in a while, but it scratched my itch! I made a small update so that i can customize it to my use case.

Thanks for making this!

-- Adam

File: sampler/twitter-server.go [hunk 1]
```diff
@@ -18,8 +19,13 @@ var (
 	jvmStat   = regexp.MustCompile(`^jvm/uptime$`) // just uptime, for now
 )
 
-func NewTwitterServerSampler(metricsURL string) *TwitterServerSampler {
-	return &TwitterServerSampler{metricsURL}
+func NewTwitterServerSampler(metricsURL, matchRegexp string) *TwitterServerSampler {
+	var matcher *regexp.Regexp = nil
```
Comment (klingerf) [inline sampler/twitter-server.go]: Tiny nit: nil is already the zero value for pointers, so you could write this as:

```
var matcher *regexp.Regexp
```

**Review by klingerf — COMMENTED** [review]
This is great, thanks for contributing @aschepis! Would you remind reverting the imports changed from `CloudHealth` back to `klingerf`? Once that's done I'd be happy to merge.

**Comment by aschepis** [comment]
@klingerf -- should be all set!

**Review by klingerf — APPROVED** [review]
This is great, thanks for contributing!

**Comment by aschepis** [comment]
happy to help.  i'll switch my containers back to using the docker hub container as soon as its updated!

**Comment by klingerf** [comment]
@aschepis pushed! and i upgrade the image to go-1.8. let me know if you run into any issues.

**Pull request merged**